src/Logary.Facade/Facade.fs

```diff
@@ -684,6 +703,55 @@ module internal Formatting =
     formatExn message.fields +
     formatFields matchedFields message.fields
 
+/// Assists with implementing a custom tokeniser for the `LiterateConsoleTarget`.
+module LiterateFormatting =
```

Comment:
Well, the extension point is in the `LiterateConsoleTarget` constructor already (`literateTokeniser`), but it's practically impossible to use it right now, because everything is internal. Making this public is intentional.